src/realtime/flood/tests/test_flood_hazard.py

```diff
@@ -115,6 +115,15 @@ def test_geojson_conversion(self):
 
         self.assertDictContainsSubset(expected_keywords, actual_keywords)
 
+        expected_extra_keywords = {
+            u'flood_event_time': u'2017-02-21T05:00:00',
+            u'flood_event_id': u'2017022105-6-rw',
+            u'time_zone': u'Asia/Jakarta'
```

Comment:
Use dictionary key? i.e. `extra_keyword_time_zone['key']`

Reply:
No, it was intentional by me for the unittests. I think it is useful when we had changes from desktop, the unittests will alert us of potential problems. Because it maybe that other service will use the keywords result directly because they have no access to InaSAFE definition. For example in GeoSAFE.

Reply:
ah yes, it's unit test. I did it also in inasafe-django